sdk/eventhub/azure-eventhub/tests/conftest.py

```diff
@@ -110,16 +110,38 @@ def timeout_factor(uamqp_transport):
 def fake_span():
     return FakeSpan
 
+
+@pytest.fixture(scope="session")
+def get_credential():
+    use_pwsh = os.environ.get("AZURE_TEST_USE_PWSH_AUTH", "false")
+    use_cli = os.environ.get("AZURE_TEST_USE_CLI_AUTH", "false")
+
+    credential = EnvironmentCredential()
```

Comment:
I would consider changing up the logic here to maybe `return` the credential immediately from each conditional branch, and if neither PWSH or CLI are requested, returning the `EnvironmentCredential`. I say that because initializing the `EnvironmentCredential` may raise an error if you don't have necessary environment variables set, even if user auth would still work in that setup

Reply:
updated!